packages/api-form-builder-so-ddb-es/src/configurations.ts

```diff
@@ -54,7 +54,10 @@ export const configurations: Configurations = {
             .join("-")
             .toLowerCase();
 
-        const prefix = process.env.ELASTIC_SEARCH_INDEX_PREFIX || "";
+        const prefix =
```

Comment:
Maybe we do the same thing e did here?

<img width="1169" alt="image" src="https://github.com/user-attachments/assets/4e6aece7-4d40-4df7-9a52-ffa95dd325d2">